src/components/statusCard/styles/StatusCard.scss

```diff
@@ -0,0 +1,56 @@
+@import '~coveo-styleguide/scss/common/palette.scss';
```

Comment:
can we create an alias with this 3 imports ? if possible to do that. I think we will use these imports in many files

Reply:
I don't think this is possible. I didn't need the map anymore and we don't always want to import both other files.